src/createServer.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const express = require('express');
+const cors = require('cors');
+
+const usersRouter = require('./routes/users');
+const expensesRouter = require('./routes/expenses');
+
+function createServer() {
+  const app = express();
+
+  app.use(cors());
+  app.use('/users', express.json(), usersRouter);
```

Comment:
Move `express.json()` calls to a separate `app.use(express.json())` so you don't repeat it